roman_photoz/logger.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+
+def setup_logging(log_file="roman_photoz.log", level=logging.INFO):
+    """Set up logging configuration for roman_photoz module"""
+    # Create a logger specific to our module instead of configuring the root logger
+    logger = logging.getLogger("roman_photoz")
+
+    # Only configure if it hasn't been configured yet
+    if not logger.handlers:
+        # Set propagate to False to prevent messages from being passed to the root logger
+        logger.propagate = False
+
+        # Set the level
+        logger.setLevel(level)
+
+        # Create formatter
+        formatter = logging.Formatter(
+            "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+        )
+
+        # Create handlers
+        console_handler = logging.StreamHandler()
+        console_handler.setFormatter(formatter)
+
+        file_handler = logging.FileHandler(log_file, mode="a")
```

Comment:
This means we'll start making a log file every time we run?  I don't think any of our other tools do that?  If we wanted to follow the example of our other tools, putting a log as an extension in the output product would make a lot of sense.

Reply:
Yeah, although it is common practice for web apps, I don't think this is a good idea for this project. ;)

Reply:
Removed the option to log out to a file.